### PR TITLE
Core 8591 Allow repeating template fields-Refactor code based on PR Comments 

### PR DESCRIPTION
**Pull request by sri2k1us**
- Change Accordion layout to Tabs
- Split view and Dialog
- DI and code splits
- Fix bugs

### CODE REVIEW

File: de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/MetadataDialog.java [hunk 1]
```diff
@@ -56,36 +54,36 @@ public void onDialogHide(DialogHideEvent event) {
         }
     }
 
-    private final MetadataView.Presenter meta_pre;
-    private final MetadataView metadataView;
+    MetadataView.Presenter meta_pre;
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/MetadataDialog.java]: We should probably change this to camelCase.

File: de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/MetadataDialog.java [hunk 2]
```diff
@@ -34,16 +31,17 @@ public void onSelect(SelectEvent event) {
     private class OkSelectHandler implements SelectEvent.SelectHandler {
         @Override
         public void onSelect(SelectEvent event) {
-            if (!metadataView.isValid()) {
-                ConfirmMessageBox cmb = new ConfirmMessageBox(I18N.DISPLAY.error(),
-                                                              appearance.metadataSaveError());
+            if (!meta_pre.isValid()) {
+                ConfirmMessageBox cmb =
+                        new ConfirmMessageBox(I18N.DISPLAY.error(), appearance.metadataSaveError());
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/MetadataDialog.java]: Minor thing, but I've been trying to get rid of these I18N references and using appearance classes  instead.

File: de-lib/src/main/java/org/iplantc/de/admin/desktop/client/gin/BelphegorAppsGinModule.java [hunk 3]
```diff
@@ -30,17 +30,15 @@
 import org.iplantc.de.admin.desktop.client.toolRequest.view.ToolRequestViewImpl;
 import org.iplantc.de.admin.desktop.client.views.BelphegorView;
 import org.iplantc.de.admin.desktop.client.views.BelphegorViewImpl;
-import org.iplantc.de.admin.desktop.client.workshopAdmin.WorkshopAdminView;
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/admin/desktop/client/gin/BelphegorAppsGinModule.java]: Were these imports supposed to be removed?

File: de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/ManageMetadataDialog.java [hunk 4]
```diff
@@ -28,20 +24,21 @@
...
     private boolean writable;
     private DiskResource resource;
     private boolean canHide;
 
+    MetadataView.Presenter mdPresenter;
+
     @Inject
-    ManageMetadataDialog(final DiskResourceServiceFacade diskResourceService,
+    ManageMetadataDialog(MetadataView.Presenter mdPresenter,
+                         final DiskResourceServiceFacade diskResourceService,
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/ManageMetadataDialog.java]: I don't believe the service facade is actually being used in the dialog - can probably remove it.

File: de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/PermanentIdRequestViewImpl.java [hunk 5]
```diff
@@ -260,8 +266,18 @@ void onSelectionChange(SelectionChangedEvent<PermanentIdRequest> event) {
     public void fetchMetadata(Folder selectedFolder,
                               PermanentIdRequestPresenterAppearance appearance,
                               DiskResourceServiceFacade drsvc) {
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/PermanentIdRequestViewImpl.java]: The service facade isn't used here anymore, we can probably remove this.

File: de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/MetadataDialog.java (hunk 2 above)
Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/MetadataDialog.java]: 👍

File: de-lib/src/main/java/org/iplantc/de/admin/desktop/client/gin/BelphegorAppsGinModule.java (hunk 3 above)
Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/admin/desktop/client/gin/BelphegorAppsGinModule.java]: I was messing with this class. I should have removed inadvertently. But Looks like they are not used anymore.

File: de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/PermanentIdRequestViewImpl.java (hunk 5 above)
Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/PermanentIdRequestViewImpl.java]: 👍

**Comment by sri2k1us** [comment]
Thanks for the feedback.

**Pull request merged**